public/pages/Snapshots/containers/Snapshots/Snapshots.tsx

```diff
@@ -226,7 +218,8 @@ export default class Snapshots extends Component<SnapshotsProps, SnapshotsState>
       const { snapshotManagementService } = this.props;
       const response = await snapshotManagementService.restoreSnapshot(snapshotId, repository, options);
       if (response.ok) {
-        this.context.notifications.toasts.addSuccess(`Restored snapshot ${snapshotId} to repository ${repository}.`);
+        console.log(response);
```

Comment:
Remove

Reply:
Can do now, is most definitely addressed in next PRs as well.

Reply:
Done